devicetypes/smartthings/zwave-lock.src/zwave-lock.groovy

```diff
@@ -37,12 +37,15 @@ metadata {
 		fingerprint mfr:"003B", prod:"6341", model:"0544", deviceJoinName: "Schlage Camelot Touchscreen Deadbolt Door Lock"
 		fingerprint mfr:"003B", prod:"6341", model:"5044", deviceJoinName: "Schlage Century Touchscreen Deadbolt Door Lock"
 		fingerprint mfr:"003B", prod:"634B", model:"504C", deviceJoinName: "Schlage Connected Keypad Lever Door Lock"
-		fingerprint mfr:"0129", prod:"0002", model:"0800", deviceJoinName: "Yale Touchscreen Deadbolt Door Lock" // YRD120
-		fingerprint mfr:"0129", prod:"0002", model:"0000", deviceJoinName: "Yale Touchscreen Deadbolt Door Lock" // YRD220, YRD240
-		fingerprint mfr:"0129", prod:"0002", model:"FFFF", deviceJoinName: "Yale Touchscreen Lever Door Lock" // YRD220
-		fingerprint mfr:"0129", prod:"0004", model:"0800", deviceJoinName: "Yale Push Button Deadbolt Door Lock" // YRD110
-		fingerprint mfr:"0129", prod:"0004", model:"0000", deviceJoinName: "Yale Push Button Deadbolt Door Lock" // YRD210
-		fingerprint mfr:"0129", prod:"0001", model:"0000", deviceJoinName: "Yale Push Button Lever Door Lock" // YRD210
+		// Yale mfr could be both 0109 or 0129 for prod < 0005
+		fingerprint mfr:"0109", prod:"0001", deviceJoinName: "Yale Push Button Lever Door Lock" // YRD210
+		fingerprint mfr:"0129", prod:"0001", deviceJoinName: "Yale Push Button Lever Door Lock" // YRD210
+		fingerprint mfr:"0109", prod:"0002", deviceJoinName: "Yale Touchscreen Deadbolt Door Lock" // YRD120, YRD220, YRD240
+		fingerprint mfr:"0129", prod:"0002", deviceJoinName: "Yale Touchscreen Deadbolt Door Lock" // YRD120, YRD220, YRD240
+		fingerprint mfr:"0109", prod:"0002", deviceJoinName: "Yale Touchscreen Lever Door Lock" // YRD220
+		fingerprint mfr:"0129", prod:"0002", deviceJoinName: "Yale Touchscreen Lever Door Lock" // YRD220
+		fingerprint mfr:"0109", prod:"0004", deviceJoinName: "Yale Push Button Deadbolt Door Lock" // YRD110, YRD210
+		fingerprint mfr:"0129", prod:"0004", deviceJoinName: "Yale Push Button Deadbolt Door Lock" // YRD110, YRD210
 		fingerprint mfr:"0129", prod:"8002", model:"0600", deviceJoinName: "Yale Assure Lock with Bluetooth"
```

Comment:
This line would not pair the Assure T1L that I have, because the model of my device is `1600`, and mine was purchased through the SmartThings store. Please remove the model for now. I don't know if the `0600` is a typo or if `0600` is a product id in addition to `1600`

This is my Assure lock's raw description:
`zw:Fs type:4003 mfr:0129 prod:8002 model:1600 ver:1.74 zwv:4.24 lib:03 cc:5E,72,98,5A,73,86 sec:80,62,85,59,71,70,63,8A,8B,4C,4E,7A role:07 ff:8300 ui:8300`